apps/main/src/loan/components/PageLlamaMarkets/filters/RangeSliderFilter.tsx

```diff
@@ -16,7 +16,7 @@ import { cleanColumnId } from '@ui-kit/shared/ui/DataTable/TableVisibilitySettin
 const getMaxValueFromData = <T extends any, K extends DeepKeys<T>>(data: T[], field: K) =>
   data.reduce((acc, item) => Math.max(acc, get(item, field) as number), 0)
 
-type Range = [number, number]
+type Range<T = number> = [T, T]
```

Comment:
Curious to know why you made this generic. I don't see any case where you're using the range type where `T != number`

Reply:
Yeah it's not really used in this project yet. It's just to make it more reusable, but I'll revert this in https://github.com/curvefi/curve-frontend/pull/849